zowe/zowe-explorer-vscode
# fix: formatting of 401 error message

## Proposed changes

This won't need ported for 3.0.3 as this was a side effect of the error correlator


### How to test

Same steps as in #3302:

- Checkout this branch in the repo with VS Code opened
- `pnpm install && pnpm build`
- After the build has finished, click on the Run & Debug panel on the VS Code Side Bar
- Select "Run VS Code Extension" from the tasks list at the top and select the Play ▶ button
- Once opened, make sure that you have (or set) valid credentials on a profile in ZE
- Execute a search on that profile and expand a folder so that a file/PDS member/spool item is visible
- Before opening the file, right click on your profile -> "Manage Profile" -> "Update Credentials"
- Enter in invalid credentials. I recommend using a fake username to prevent lockouts
- Once finished, try to open the file. Notice the error in the editor appears but you are prompted to update your credentials.

Except this time, you'll see a nicer error message 😋

## Release Notes

Milestone: 3.0.3

Changelog: No changelog since fix hasn't been published yet

## Types of changes




## Checklist



**General**

  - **v2 and below:** `yarn workspace vscode-extension-for-zowe vscode:prepublish`
  - **v3:** `pnpm --filter vscode-extension-for-zowe vscode:prepublish`

**Code coverage**


**Deployment**

  - If you're an outside contributor, please post in the [#zowe-doc Slack channel](https://openmainframeproject.slack.com/archives/CC961JYMQ) to coordinate documentation.
  - Otherwise, please check with the rest of the squad about any needed documentation before merging.

## Further comments
